modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6006,6 +6006,7 @@ dynamically-loaded-data=Dynamically Loaded Data
 e=%e
 each-editable-image-element-must-contain-an-img-tag=Each editable image element must contain an {0}IMG{1} tag.
 each-editable-link-element-must-contain-an-a-tag=Each editable link element must contain an {0}A{1} tag.
+each-email-recipient-listed-on-the-to-will-be-sent-separately-and-not-be-able-to-see-others-in-copy=Each email recipient listed on the "To" will be sent separately and not be able to see others in copy.
```

Comment:
@seltong I'm thinking we're missing one word in the sentence. Shouldn't it be "...listed on the "To" field" or something similar? Could you confirm with the designer, please?

Reply:
Sure, I will confirm

Reply:
Waiting for reply